From 425354918f774c39c68b79de55691ee262cdcb84 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Wed, 25 Mar 2015 17:57:05 +0100
Subject: [PATCH] Build from lib
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Building from lib.modern adds "modern::" prefix to manual pages.

This is a work-around for CPAN RT#103116. Creating symlink is not
enough for EU::MM.

Signed-off-by: Petr Písař <ppisar@redhat.com>

diff --git a/Makefile.PL b/Makefile.PL
index a8dbc7e..1b5b51a 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -1,10 +1,15 @@
 
 use ExtUtils::MakeMaker;
 
+if (!-d 'lib') {
+    rename (($] > 5.010 ? 'lib.modern' : 'lib.old'), 'lib') or
+        die "Could not rename to lib: $!";
+}
+
 WriteMakefile( 
 	'NAME'		=> 'Text-Wrap',
 	'DISTNAME'	  => 'Text-Tabs+Wrap',
-	'VERSION_FROM' => 'lib.modern/Text/Wrap.pm',
+	'VERSION_FROM' => 'lib/Text/Wrap.pm',
 	($] >= 5.005 ? 
 		(
 			ABSTRACT	=> 'Expand tabs and do simple line wrapping',
@@ -21,16 +26,5 @@ WriteMakefile(
 	'dist' => { COMPRESS=>"gzip", SUFFIX=>"gz" },
 	'INSTALLDIRS'	=> 'perl',
 
-	PM =>	($] > 5.010 ?
-		{
-			'lib.modern/Text/Tabs.pm'	=> '$(INST_LIBDIR)/Text/Tabs.pm',
-			'lib.modern/Text/Wrap.pm'	=> '$(INST_LIBDIR)/Text/Wrap.pm',
-		}
-		:
-		{
-			'lib.old/Text/Tabs.pm'	=> '$(INST_LIBDIR)/Text/Tabs.pm',
-			'lib.old/Text/Wrap.pm'	=> '$(INST_LIBDIR)/Text/Wrap.pm',
-		})
-
 	);
 
-- 
2.1.0

